src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -883,6 +884,14 @@ private async Task<ConnectBlocksResult> ConnectBlockAsync(ChainedHeaderBlock blo
                     PeersToBan = badPeers
                 };
 
+                if (validationContext.RejectUntil != null)
```

Comment:
`// If set, use the block reject until time as the ban duration instead of the default 10 minute ban.`

Reply:
Added